openapi.yaml

```diff
@@ -586,12 +639,15 @@ components:
       format: date
       nullable: true
       description: Returned date of the book
-      example: null
+      example: '2023-08-30'
     BorrowStatus:
       type: string
       nullable: true
       description: Current borrow status of the book (ongoing/returned/overdue)
-      example: ongoing
+      enum:
+        - ongoing
+        - returned
+        - overdue
```

Comment:
This enum is actually the same as `filter[status]`'s enum. You should be able to use a reference for it.

Reply:
done for both books and members